awx/ui_next/src/components/AppContainer/AppContainer.jsx

```diff
@@ -138,6 +139,13 @@ function AppContainer({ navRouteConfig = [], children }) {
     }
   }, [handleLogout, timeRemaining]);
 
+  useEffect(() => {
+    if ('analytics_status' in config) {
+      issuePendoIdentity(config, 'foobar');
```

Comment:
Need to pull the real pendo key and pass it along here

Reply:
Actually, I forgot that I pushed the logic to grab the api key down to the utility function so this extra param is not needed.